### ISSUE TITLE
Define a "defaiult" folder for the font files and remember the last used

### ISSUE THREAD

**Issue by Pairan** [avsaase/walksnail-osd-tool#6]
### Is there an existing feature request for this?

- [X] I have searched the existing open and closed issues

### Feature description

It would be nice if the encoder would remember where the last font file came from and perhaps have it in a default folder.

**Comment by avsaase** [comment]
Good idea. I was also thinking of including a set of font files so you can select one from a dropdown menu. I would need to get permission from the font file authors to do that.

**Comment by Pairan** [comment]
That of course would make it very easy

**Comment by dz0ny** [comment]
For starters you can use the default font https://drive.google.com/file/d/1c3CRgXYQaM3Tt4ukLSIvoogScQZs9w49/view
